docs/src/data/frameworks.ts

```diff
@@ -8,32 +6,30 @@ export type Framework =
   | 'vue';
 export type Frameworks = Framework[];
 
-export const FRAMEWORKS: Frameworks = IS_REACT_NATIVE_ENABLED
-  ? ['angular', 'flutter', 'react', 'react-native', 'vue']
-  : ['angular', 'flutter', 'react', 'vue'];
+export const FRAMEWORKS: Frameworks = [
+  'angular',
+  'flutter',
+  'react',
+  'react-native',
+  'vue',
+];
 
-const BASE_FRAMEWORK_DISPLAY_NAMES = {
+export const FRAMEWORK_DISPLAY_NAMES: Record<Framework, string> = {
   angular: 'Angular',
   flutter: 'Flutter',
   react: 'React',
+  'react-native': 'React Native',
   vue: 'Vue',
 };
 
-export const FRAMEWORK_DISPLAY_NAMES = IS_REACT_NATIVE_ENABLED
-  ? {
-      ...BASE_FRAMEWORK_DISPLAY_NAMES,
-      'react-native': 'React Native',
-    }
-  : BASE_FRAMEWORK_DISPLAY_NAMES;
-
 // React Native requires direct installation of dependencies with native modules
 export const REACT_NATIVE_DEPENDENCIES =
-  '@react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill react-native-safe-area-context';
+  'react-native-safe-area-context @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill';
```

Comment:
Can we introduce some line breaks here? they don't all fit on the screen and there is no indication of there being more dependencies (no ellipsis or anything)